config/serialization.go

```diff
@@ -31,83 +30,96 @@ func configFile() string {
 // Load will load confiration file, and initialize a default configuration
 // if no configuration is present. This only returns an error if a configuration
 // file is present, but we are unable to parse it.
+// TODO	we could simplify this function and only go through the definitions once
+//		and what happens if a value is tied to an env var AND to the config file?
```

Comment:
Opened issue #105.

I feel we should deal with this post-merge. Currently, the code I wrote has the same behaviour as the old code, and I don't think it's useful to change it until we have a definitive answer.

It's also a question that's not related to cobra so it should be a different branch and PR.